frontend/modelarchive/src/main/java/org/pytorch/serve/archive/ModelArchive.java

```diff
@@ -167,8 +167,14 @@ public void validate() throws InvalidModelException {
                 throw new InvalidModelException("Runtime is not defined or invalid.");
             }
 
-            if (manifest.getEngine() != null && manifest.getEngine().getEngineName() == null) {
-                throw new InvalidModelException("engineName is required in <engine>.");
+            if (manifest.getArchiverVersion() == null) {
```

Comment:
@harshbafna  Will people need to recreate their .mar files for existing models after this change? Want to understand the implications from migration perspective

Reply:
@chauhang : Per current changes, users don't need to re-create their existing .mar files. However, added a warning to migrate to the latest version.

The sanity script and regression-suite execute successfully, which ensures that the old mar files from model-zoo work without any issues.